stdlib/public/core/Arrays.swift.gyb

```diff
@@ -508,7 +508,7 @@ extension ${Self} : _ArrayType {
 
   /// Construct a ${Self} of `count` elements, each initialized to
   /// `repeatedValue`.
-  @swift3_migration(renamed="init(repeating:count:)")
+  @swift3_migration(message="Renamed to init(repeating:count:) and reordered parameters")
```

Comment:
I thought `renamed=""` could handle parameter reordering.  If it can't, ignore this comment.